.github/workflows/prerelease.yml

```diff
@@ -40,6 +50,12 @@ jobs:
           - arm64
           - x86_64
     steps:
+      # Note that a duplicate of this step must be added at the top of
+      # each job.
```

Comment:
```suggestion
```

Reply:
Resolved in [5a02fb8](https://github.com/cisagov/skeleton-packer/pull/376/commits/5a02fb8244d9bfb8d1a37cd1e2e3deb60bdf03cc).